bundles/org.openhab.binding.groupepsa/src/main/java/org/openhab/binding/groupepsa/internal/bridge/GroupePSABridgeHandler.java

```diff
@@ -0,0 +1,214 @@
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.groupepsa.internal.bridge;
+
+import static org.openhab.binding.groupepsa.internal.GroupePSABindingConstants.THING_TYPE_BRIDGE;
+import static org.openhab.binding.groupepsa.internal.GroupePSABindingConstants.VendorConstants;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.client.HttpClient;
+import org.openhab.binding.groupepsa.internal.rest.api.GroupePSAConnectApi;
+import org.openhab.binding.groupepsa.internal.rest.api.dto.Vehicle;
+import org.openhab.binding.groupepsa.internal.rest.api.dto.VehicleStatus;
+import org.openhab.binding.groupepsa.internal.rest.exceptions.GroupePSACommunicationException;
+import org.openhab.core.auth.client.oauth2.AccessTokenResponse;
+import org.openhab.core.auth.client.oauth2.OAuthClientService;
+import org.openhab.core.auth.client.oauth2.OAuthException;
+import org.openhab.core.auth.client.oauth2.OAuthFactory;
+import org.openhab.core.auth.client.oauth2.OAuthResponseException;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.ThingTypeUID;
+import org.openhab.core.thing.binding.BaseBridgeHandler;
+import org.openhab.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link GroupePSABridgeHandler} is responsible for handling commands,
+ * which are sent to one of the channels.
+ *
+ * @author Arjan Mels - Initial contribution
+ */
+@NonNullByDefault
+public class GroupePSABridgeHandler extends BaseBridgeHandler {
+    private final Logger logger = LoggerFactory.getLogger(GroupePSABridgeHandler.class);
+
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Collections.singleton(THING_TYPE_BRIDGE);
+    private static final long DEFAULT_POLLING_INTERVAL_M = TimeUnit.HOURS.toMinutes(1);
+
+    private final OAuthFactory oAuthFactory;
+
+    private @NonNullByDefault({}) OAuthClientService oAuthService;
+    private @Nullable ScheduledFuture<?> groupepsaBridgePollingJob;
+    private final HttpClient httpClient;
+
+    private @NonNullByDefault({}) String vendor;
+    private @NonNullByDefault({}) VendorConstants vendorConstants;
+    private @NonNullByDefault({}) String userName;
+    private @NonNullByDefault({}) String password;
+    private @NonNullByDefault({}) String clientId;
+    private @NonNullByDefault({}) String clientSecret;
+
+    private @NonNullByDefault({}) GroupePSAConnectApi groupePSAApi;
```

Comment:
This should be `@Nullable`. Disabling the null annotations with `@NonNullByDefault({}` should be used only if you have good reasons. Please check all.